bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/model/CustomResponseTransformer.java

```diff
@@ -82,24 +82,9 @@ private void updateChargerStartStop(Map<Channel, State> result, String value, Js
         Channel channel = channelProvider.getChannel(CHANNEL_GROUP_CHARGER_COMMANDS, CHANNEL_CHARGER_START_STOP);
         if (channel != null) {
             int val = Integer.parseInt(value);
-            // state >= 3 && state < 7 will mean charging, ready to charge or charging finished
-            boolean charging = val >= CHARGER_OP_STATE_CHARGING && val < CHARGER_OP_STATE_NOT_AUTHENTICATED;
-
-            String rfnc = Utils.getAsString(rawData, CHANNEL_CHARGER_REASON_FOR_NO_CURRENT);
-            int reasonForNoCurrent = Integer.valueOf(rfnc == null ? "-1" : rfnc);
-            boolean paused = false;
-            if (val == CHARGER_OP_STATE_WAITING) {
-                switch (reasonForNoCurrent) {
-                    case CHARGER_REASON_FOR_NO_CURRENT_CHARGER_LIMIT:
-                    case CHARGER_REASON_FOR_NO_CURRENT_CIRCUIT_LIMIT:
-                        paused = true;
-                        break;
-                    default:
-                        paused = false;
-                        break;
-                }
-            }
-            result.put(channel, OnOffType.from(charging || paused));
+            // 2 <= state < 7 will mean car is connected and session is authenticated
+            boolean authenticated = val >= CHARGER_OP_STATE_WAITING && val < CHARGER_OP_STATE_NOT_AUTHENTICATED;
+            result.put(channel, OnOffType.from(authenticated));
```

Comment:
The comment and the condition make it not super simple to read, maybe this is more clear. Just a suggestion.
```suggestion
            boolean connectedAndAuthenticated = state >= CHARGER_OP_STATE_WAITING && state < CHARGER_OP_STATE_NOT_AUTHENTICATED;
            result.put(channel, OnOffType.from(connectedAndAuthenticated ));
```